core/src/test/java/nu/yona/server/device/service/DeviceServiceTest.java

```diff
@@ -169,6 +176,8 @@ public void addDeviceToUser_addFirstDevice_userHasOneDevice()
 
 		UserDevice device = devices.iterator().next();
 		assertDevice(device, startTime, deviceName, operatingSystem, 0);
+
+		assertVpnAccountCreated(device);
```

Comment:
We could move those new asserts to separate tests, so that the tests test one thing and stay easy to read.

Reply:
I don't favor that. It's an assertion that belongs to this test, not a new test.

Reply:
Actually, the name of the test should then change to `_userHasOneDeviceWithVpnAccountCreated`

Reply:
What about ``_deviceProperlyAdded``? Encoding all assertions in that text seems overkill to me.